### DISCUSSION TITLE
Further Inkscape SVG integration

### DISCUSSION THREAD

**Discussion by SpeeQz1** [friction2d/friction#309] [Ideas]
Currently when trying to import SVGs from inkscape with a Pattern infil and a circle clip applied to them, Friction seems to properly import the clipped objects but the pattern is not copied over and the object is filled with black colour. Same applies for the other fills like Mesh gradient. The regular fill has the appropriate colors copied over.

![image](https://github.com/user-attachments/assets/ec8df796-0b74-48e9-8ba6-8752d1648931)

![image](https://github.com/user-attachments/assets/a952f88e-b37a-4df6-b14b-d21ca9aabc9c)

**Comment by rodlie** [comment]
Friction has no concept of patterns, but it would be possible to implement it at some point in the future.

Regarding mesh, it might be possible to implement, unsure. But it's not supported by browsers, so it has no value for Friction.
